VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:

BO_ 1 Message_1: 8 Vector__XXX
 SG_ Signal_1 : 0|8@1- (1,0) [0|0] "" Vector__XXX
 
BO_ 2 Message_2: 8 Vector__XXX
 SG_ Signal_2 : 0|8@1- (1,0) [0|0] "" Vector__XXX
BO_ 3 Message_3: 8 Vector__XXX
 SG_ Signal_3 : 0|8@1- (1,0) [0|0] "" Vector__XXX

CM_ SG_ 1 Signal_1 "Comment
";
VAL_ 2 Signal_2 1 " Error B" 0 " Error A" ;
